Sources/Calendar.php

```diff
@@ -365,6 +423,55 @@ function CalendarPost()
 	$context['linktree'][] = array(
 		'name' => $context['page_title'],
 	);
+
+	loadCSSFile('jquery-ui.datepicker.css', array('defer' => false), 'smf_datepicker');
+	loadCSSFile('jquery.timepicker.css', array('defer' => false), 'smf_timepicker');
+	loadJavascriptFile('jquery-ui.datepicker.min.js', array('defer' => true), 'smf_datepicker');
```

Comment:
That would be a very nice addition...I might actually throw in a PR for this functionality. Definitely unappealing to have to make multiple calls to a ~~method~~ *function* instead of passing in an array.

And actually, it is Java__S__cript: https://en.wikipedia.org/wiki/JavaScript